Doc/whatsnew/3.10.rst

```diff
@@ -1948,6 +1948,12 @@ Changes in the C API
        source_buf = PyBytes_AsString(source_bytes_object);
        code = Py_CompileString(source_buf, filename, Py_file_input);
 
+  * For ``FrameObject`` objects, the ``f_lasti`` member now represents a wordcode
```

Comment:
Can you add a note to use PyFrame_GetLineNumber() instead

Reply:
Fixed in https://github.com/python/cpython/pull/28208